tutorials/flux_discretizations.ipynb

```diff
@@ -134,7 +134,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        "source": f, 
</pre><p>FYI, we have not enforced 88 character line length in tutorials.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>